erpnext/accounts/report/trial_balance/trial_balance.py

```diff
@@ -87,8 +87,7 @@ def get_opening_balances(filters):
 
 
 def get_rootwise_opening_balances(filters, report_type):
-	additional_conditions = " and posting_date >= %(year_start_date)s" \
-		if report_type == "Profit and Loss" else ""
+	additional_conditions = ""
```

Comment:
I think it is not a standard expectation, users can be confused, if previous year's balance is included in P&L opening balance.

Instead, may be we can show last year's profit/loss in a separate row, just like Balance Sheet, it will be much clearer.